linera-service/src/proxy/grpc.rs

```diff
@@ -497,7 +498,7 @@ where
             .await
             .map_err(|err| Status::from_error(Box::new(err)))?
```

Comment:
Actually, a malicious client could still cause a lot of I/O here. Maybe we should also chunk the batches of certificates that we're reading from storage into memory at a time?

Reply:
How many certificates do you suggest to read at a time?

Reply:
Maybe 100? We could make that a constant for now, and create an issue for measuring and optimizing it later.

Reply:
Another approach (that might be too slow) is to read the certificates from storage one by one until we're at the `GRPC_CHUNKED_MESSAGE_FILL_LIMIT`, then just return. Picking a constant number of certificates might be faster, but will require some iterating to find an optimal constant.
We could also do the slow approach, log how many certificates we're being usually able to fill within the limit, then optimize further with a constant number of certificates to read from storage at once based on that data.

Reply:
Lifted to an [issue](https://github.com/linera-io/linera-protocol/issues/2658) and will tackle in the next PR.